baseplate/experiments/__init__.py

```diff
@@ -90,6 +97,18 @@ def _get_experiment(self, name):
             self._experiment_cache[name] = experiment
         return self._experiment_cache[name]
 
+    def _get_all_experiment_names(self):
+        config = self._config_watcher.get_data()
+        exp_names = []
+        if config:
+            for name, cfg in config.items():
```

Comment:
this could be written `exp_names = list(config.keys())`

i'd also suggest not using an abbreviated variable name like that (per [pep-8](https://github.com/reddit/styleguide/blob/master/python/pep-0008.rst))

Reply:
fixed